PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,11 +2,5 @@
 
 <!-- Why is this PR necessary? If available, include links to a JIRA ticket or other relevant documentation. -->
 
-### Changes
```

Comment:
Why is this removed?

Reply:
Copied over the one from: https://github.com/transferwise/wise-github-bot/blob/master/.github/PULL_REQUEST_TEMPLATE.md

I don't know what went into defining it as such in the `wise-github-bot` repo, but my thoughts on this are that it may be unnecessary and somewhat duplicate of the other main header. Given that it is a **sub-header**, I find people are removing/modifying the sub-header at their will either way (renaming it to Changelog, adding more sub-headers, etc.) to supplement their message under the main header.

Reply:
Makes sense, thanks! Agree it doesn't make sense as a **sub**header indeed and that we should align.